docs/Tutorials/Configure-External-Signer-TLS.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Configure TLS communication with an external signer
+---
+
+# Configure TLS communication
+
+Configure TLS communication with an external signer such as [Web3Signer] which accepts
+connections from clients that use trusted CA certificates or self-signed certificates.
+
+This tutorial configures TLS between Teku and Web3Signer, and use the
+[`keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html) utility to generate
+keystores and the truststore that contain self-signed certificates.
+
+!!! info
+
+    `keytool` is available with the JDK installation, you can also use OpenSSL.
```

Comment:
keytool is available with both JRE/JDK installation

Reply:
Done